cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/workflow/WorkflowDriver.java

```diff
@@ -427,14 +446,39 @@ private void executeCondition(ApplicationSpecification appSpec, WorkflowConditio
     executeAll(iterator, appSpec, instantiator, classLoader, token);
   }
 
+  private void createLocalDatasets() throws IOException, DatasetManagementException {
+    for (Map.Entry<String, DatasetCreationSpec> instanceEntry : workflowSpec.getLocalDatasetSpecs().entrySet()) {
+      String localInstanceName = getLocalDatasetName(instanceEntry.getKey(), runId.getId());
+      Id.DatasetInstance instanceId = Id.DatasetInstance.from(program.getNamespaceId(), localInstanceName);
+      DatasetCreationSpec instanceSpec = instanceEntry.getValue();
+      LOG.info("Adding Workflow local dataset instance: {}", localInstanceName);
```

Comment:
Since it only gets printed once I thought its better to have it as `INFO` so that we know what local datasets are getting created. However realized that dataset framework also prints the INFO log for the same. Moved to `DEBUG`